test/functional/tests/fault_injection/test_standby_activate_different_cache_path.py

```diff
@@ -0,0 +1,94 @@
+#
+# Copyright(c) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+import pytest
+
+from api.cas import casadm
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from test_utils.size import Size, Unit
+from test_tools.dd import Dd
+from api.cas.cache_config import CacheStatus
+from api.cas.cache_config import CacheLineSize
+from api.cas.cli import standby_activate_cmd
+from api.cas.casadm_parser import get_caches
+
+cache_size = Size(16, Unit.GibiByte)
+block_size = Size(1, Unit.Blocks512)
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.nand, DiskType.optane]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+@pytest.mark.parametrizex("cache_line_size", CacheLineSize)
+def test_standby_activate_different_cache_path(cache_line_size):
+    """
+    title: test_standby_activate_different_cache_path
+    description: |
+        Symlink-activated CAS partitions distinction test.
+    pass_criteria:
+      - Prepare two partitions with the same size, start and stop cache instance on one.
+      - Successfully start a standby cache with valid CAS metadata from the first partition.
+      - Activate cache providing symbolic link which points to second partition.
+      - Verify that the activation succeeded by checking length of array with caches.
+      - Verify that the regular cache instance is running by checking its state.
```

Comment:
At this moment pass criteria remind test steps. Consider which criteria have to be satisfied for test to have value and how to check them.